src/main/java/org/dinosaur/foodbowl/domain/review/application/ReviewService.java

```diff
@@ -68,4 +72,20 @@ private StoreCreateDto convertStoreCreateDto(ReviewCreateRequest reviewCreateReq
                 reviewCreateRequest.schoolY()
         );
     }
+
+    @Transactional
+    public void delete(Long id, Member member) {
+        Review review = reviewRepository.findById(id)
+                .orElseThrow(() -> new BadRequestException(NOT_FOUND_ERROR));
+
+        if (review.isNotOwnerOf(member)) {
```

Comment:
```suggestion
        if (!review.isOwnerOf(member)) {
```
긍정문 메서드로 만들고 `!`를 붙이는 방법도 있는데, 부정 메서드를 선택하신 이유가 궁금합니다!

Reply:
큰 이유는 없고 클린 코드에 대해서 학습할 때 `!`는 최대한 자제하는 방향으로 배워서 `!`를 최대한 자제하고 있습니다 !